.github/workflows/python-package.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Build and Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10", "pypy-3.8"]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
```

Comment:
The recommended way to run `pip` these days is `python -m pip`, which makes sure you're using the same version of Python for pip and when running python programs. See e.g. https://pip.pypa.io/en/stable/user_guide/. The `pip` command normally works, but causes headaches when `pip install foo` installs `foo` for a version of python and `python my-script.py` runs another version where `foo` isn't installed.

Let's use `python -m pip` everywhere.